README.md

```diff
@@ -198,13 +198,9 @@ For a complete list of commands, see the
 ## Building the CLI
 ### Developing
 
-Running ``godep save ./ecs-cli/...`` in the `ecs-cli/` directory creates/updates the vendor/
-directory with dependencies. Make sure that you are using go1.6 and that you have the latest godep (`go get -u github.com/tools/godep`).
-
-For developing code, the correct GOPATH can be printed by running the
-`./scripts/shared_env` script.
-
-This can be set as GOPATH on the dev box.
+* Make sure you are using go1.8 (`go version`).
+* Copy the source code (`go get github.com/aws/amazon-ecs-cli`).
```

Comment:
Add a note in parens or something to ignore the warning message from `go get`:

package github.com/aws/amazon-ecs-cli: no buildable Go source files in /Users/wppttt/temp_go/src/github.com/aws/amazon-ecs-cli